subsys/partition_manager/Kconfig

```
@@ -177,4 +177,12 @@ config PM_SRAM_SIZE
 	default $(dt_node_reg_size_hex,/soc/memory@21000000) if SOC_NRF5340_CPUNET
 	default $(dt_node_reg_size_hex,/soc/memory@20000000)
 
+
+config PM_RESERVE_HUK_KEY
+	bool "Reserve the last flash block for the HUK key"
```

People shouldn't need to read the help to know what HUK means, and we are outside security context in this menu.
```suggestion
	bool "Reserve the last flash block for the Hardware Unique Key"
```

or
```suggestion
	bool "Reserve the last flash block for the Hardware Unique Key (HUK)"
```